vendor/assets/javascripts/mediaelement/plugins/quality-avalon.js

```diff
@@ -92,16 +92,12 @@
                 value
               ) {
                 var label = void 0;
-                if (value === 'auto') {
-                  var keyExist = t.keyExist(qualityMap, value);
-                  if (keyExist) {
-                    label = value;
-                  } else {
-                    var keyValue = t.getMapIndex(qualityMap, 0);
-                    label = keyValue.key;
-                  }
-                } else {
+                var keyExist = t.keyExist(qualityMap, value);
```

Comment:
Are we changing the source code of this quality plugin?  If so, we should make a very explicit comment someplace, or override the file on our end so if we do a plugin update, it doesn't mysteriously break.

Reply:
From the name, I thought this was our override. Isn't it?

Reply:
Ah, yea sorry.  I just saw the `vendor/...` at the beginning.  I think I originally modified this for some weird focus behavior fix.